nautobot/ui/src/utils/api.js

```diff
@@ -0,0 +1,76 @@
+// Import the RTK Query methods from the React-specific entry point
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react'
+import { API_BASE, API_USER_SESSION_INFO, API_UI_MENU_INFO } from '@constants/apiPath'
+
+/*
+  The one true API!
+
+  Using Redux's RTK Query making fetching and caching a breeze! This will provide hooks and wires automagically.
```

Comment:
This follows the documented example from RTK Query: https://redux-toolkit.js.org/rtk-query/overview#basic-usage